homeassistant/components/startca/sensor.py

```diff
@@ -212,7 +212,7 @@ def bytes_to_gb(value):
     async def async_update(self):
         """Get the Start.ca bandwidth data from the web service."""
         _LOGGER.debug("Updating Start.ca usage data")
-        url = f"https://www.start.ca/support/usage/api?key={self.api_key}"
+        url = f"https://portal.start.ca/account/usage/api?key={self.api_key}"
```

Comment:
As this is a small fix to get stuff working again, I'm ok with it.

However, we cannot add more changes to this integration at this point, as it needs to be refactored. The communication with the API needs to be extracted into its own Python package hosted on PyPi.

../Frenck